kitsune/settings.py

```diff
@@ -427,33 +426,39 @@ def immutable_file_test(path, url):
     "kitsune.messages.context_processors.unread_message_count",
 ]
 
+JINJA2_TEMPLATE_OPTIONS = {
```

Comment:
I think pulling the options out makes it a little harder to read (removes them from context a little). My preference would be to leave them in `TEMPLATES `

Reply:
I agree. Let me see if I can tweak my `simplify the configuration of the extraction options` commit in a way that you'll like better, and if not we'll just dump that commit.